packages/grid/.storybook/main.js

```diff
@@ -0,0 +1,6 @@
+const storybookConfig = require( '@automattic/calypso-storybook' );
+const path = require( 'path' );
+
+module.exports = storybookConfig( {
+	stories: [ '../src/stories/**/*.stories.{js,jsx,ts,tsx}' ],
+} );
```

Comment:
I had the same question when working on the Site Admin package.
For now, I followed the existing pattern (like in components) to keep things consistent.
Also, do we want to publish the stories?

Just FYI, the following config seems to be the simpler and more conventional one in this context:
```suggestion
const storybookDefaultConfig = require( '@automattic/calypso-storybook' );
module.exports = { ...storybookDefaultConfig() };
```

Reply:
Can we use TypeScript here? It should just be a matter of renaming the file.